core/jazz_cloud-logs-streamer/_ES/apilogs.json

```diff
@@ -1,31 +1,88 @@
 {
-  "template" : "apilogs",
-  "settings" : {
-    "index.refresh_interval" : "10s"
+  "template": "apilogs",
+  "settings": {
+    "index.refresh_interval": "10s"
   },
-  "mappings" : {
-    "apilogs" : {
-       "properties" : {
-			"timestamp" : { "type":"date","format":"dateOptionalTime" },
-			"platform_log_group" : { "type":"string","index":"not_analyzed" },
-			"platform_log_stream" : { "type":"string","index":"not_analyzed" },
-			"environment" : { "type": "string", "index": "not_analyzed" },
-			"request_id" : { "type": "string", "index": "not_analyzed" },
-			"method" : { "type": "string", "index": "not_analyzed" },
-			"domain" : { "type": "string", "index": "not_analyzed" },
-			"servicename" : { "type": "string", "index": "not_analyzed" },
-			"path" : { "type": "string", "index": "not_analyzed" },
-			"user_agent" : { "type": "string", "index": "not_analyzed" },
-			"application_logs_id" : { "type": "string", "index": "not_analyzed" },
-			"origin" : { "type": "string", "index": "not_analyzed" },
-			"host" : { "type": "string", "index": "not_analyzed" },
-			"x_forwarded_port" : { "type": "string", "index": "not_analyzed" },
-			"x_forwarded_for" : { "type": "string", "index": "not_analyzed" },
-			"x_amzn_trace_id" : { "type": "string", "index": "not_analyzed" },
-			"content_type" : { "type": "string", "index": "not_analyzed" },
-			"cache_control" : { "type": "string", "index": "not_analyzed" },
-			"status" : { "type": "string", "index": "not_analyzed" }
-       }
+  "mappings": {
+    "apilogs": {
+      "properties": {
+        "timestamp": {
+          "type": "date",
+          "format": "dateOptionalTime"
+        },
+        "platform_log_group": {
+          "type": "keyword",
+          "index": "true"
```

Comment:
Hmm, it looks like value for `index` keys is boolean `true` & not string `true`. More [here](https://www.elastic.co/blog/strings-are-dead-long-live-strings).
 
------
And fields that used to be mapped as a not_analyzed string

```
{
  "foo": {
    "type" "string",
    "index": "not_analyzed"
  }
}
```

Now need to be mapped as a keyword field:

```
{
  "foo": {
    "type" "keyword",
    "index": true
  }
}
```

Reply:
Also, this file is not being used in the code/service here (which makes sense). Are we using these mappings at the installer side after we provision elasticsearch instance/container as explained [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html)?

Reply:
Yes. Validated and updated to boolean.